src/utils/auth.util.ts

```diff
@@ -0,0 +1,6 @@
+import jwt from 'jsonwebtoken';
+import config from '../config';
+
+export const getToken = function (user: any) {
+  return jwt.sign(user, config.secretKey, { expiresIn: 30 * 24 * 3600 }); // Expires in 30 days
```

Comment:
Does this need validation?
What i mean is, shouldn't this be a private route if is to get the user token?
Or is it just sign in?

Reply:
This isn't a route. It's like a helper function used to generate a signed token when the user sends a ```POST``` request at ```/api/auth/login```.
The route itself can be found at ```src/routers/auth.router.js``` and the corresponding controller is at ```src/controllers/auth.controller.js```

Reply:
Alright Thank you !